libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -142,7 +142,7 @@ export async function buildCastVoteRecordReport({
     frontNormalizedFilename: sideOneFilename,
     backNormalizedFilename: sideTwoFilename,
   } of resultSheetGenerator) {
-    const validationResult = validateSheetInterpretation([sideOne, sideTwo]);
+    const validationResult = canonicalizeSheet([sideOne, sideTwo]);
```

Comment:
rename `validationResult` to `canonicalizationResult`?

Reply:
dang, thanks